l10n_ch_import_cresus/models/account_tax.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models, fields
+
+
+class AccountTax(models.Model):
+    _inherit = "account.tax"
+
+    tax_cresus_mapping = fields.Char(
+        size=128,
```

Comment:
Just asking, do you know if the size 128 is a limitation of cresus ? if not size attribute can be dropped.